packages/terra-graphs-docs/CHANGELOG.md

```diff
@@ -20,3 +20,4 @@
 
 * Changed 
   * Moved core documentation to terra-dev-site.
+  * Changed example implementation and documentation for Combination graph to be carbon centric.
```

Comment:
nit: `Updated examples and documentation for Combination graphs to be carbon centric.`

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/181/files#diff-dc22343a9102f2186f374011efbd61a1f3ac9df233e1e9d9385065fd4e2284d1R23).